packages/terra-toolbar/src/Toolbar.jsx

```diff
@@ -36,7 +36,7 @@ const Toolbar = ({
     customProps.className,
   );
 
-  const items = React.Children.map(children, item => (<div className={cx('item')}>{item}</div>));
+  const items = React.Children.map(children, item => (item ? <div className={cx('item')}>{item}</div> : null));
```

Comment:
Does this only happen for `null` children? Does it also happen for `undefined` children?

```jsx
<Toolbar>
      {showButton && <Button text="Button 1" />}
 </Toolkit>

Reply:
Could you check if (React.Children.count(children) > 0)?
```js
let items;
if (React.Children.count(children) > 0) {
  items = React.Children.map(children, item => (<div className={cx('item')}>{item}</div>));
}
```

Reply:
or variation of above that just checks for a truthy value with a ternary
```js
const items = React.Children.count(children) ? React.Children.map(children, item => (<div className={cx('item')}>{item}</div>)) : null;
```

Reply:
> Does this only happen for `null` children? Does it also happen for `undefined` children? What happens in this scenario before/after the fix above?
 
The scenario in the snippet works fine with the changes I have made.

> Could you check if (React.Children.count(children) > 0)?

It is not working as expected with React.children.count(), reason being that the child count is displaying same even if it is a null / undefined child. So I checked directly if the child itself is a truthy value.

Reply:
Let's return 'item' instead of null here.
```suggestion
  const items = React.Children.map(children, item => (item ? <div className={cx('item')}>{item}</div> : item));
```

Reply:
Done